### PR TITLE
feat(tooling): update function signatures

### LINKED ISSUE #91: Change method signature for better partial application

### PR DESCRIPTION
**Pull request by ISO50**
- add migration information to README

close #91

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -250,17 +267,20 @@ cleanMock();
 
 ## Changelog
 
-#### `2.0.0` - **BREAKING**
+#### `authmosphere 1.0.0` - **BREAKING**
+
+Modified signature of `createAuthCodeRequestUri`, see migration guide for more information.
+
```
Comment (Retro64) [inline README.md]: maybe point out the change to a new name/version?

File: README.md [hunk 2]
```diff
@@ -250,17 +267,20 @@ cleanMock();
...
 
-#### `1.0.0` - **BREAKING**
+#### `lib-oauth-tooling 1.0.0` - **BREAKING**
 
 The signature of `requireScopesMiddleware` is now incompatible with previous versions, `precedenceFunction?` is now part of `precedenceOptions?`.
 
 ## License
 
-```
 MIT License (MIT)
```
Comment (Retro64) [inline README.md]: I know it is redundant, but maybe a link to the LICENSE file might be an opportunity?

**Comment by Retro64** [comment]
Just wondering if there are other signatures that should been updated - we should check again before merging

File: README.md (hunk 1 above)
Comment (ISO50) [inline README.md]: >> see migration guide for more information.

Is there any information missing?

**Comment by ISO50** [comment]
> Just wondering if there are other signatures that should been updated - we should check again before merging

We identfied on other, but it was an internal function. So I left it as it is.

File: README.md (hunk 1 above)
Comment (Retro64) [inline README.md]: not missing, but not explicitly visible either - maybe something like a line to mark the version break - it's just cosmetics

**Comment by Retro64** [comment]
>> Just wondering if there are other signatures that should been updated - we should check again before merging
>>
> We identfied one other, but it was an internal function. So I left it as it is.
But we should have a look at the types either - might be better to have all already identified breaking changes at once before releasing under the new name

**Comment by ISO50** [comment]
>But we should have a look at the types either - might be better to have all already identified breaking changes at once before releasing under the new name

It's already released, that why version is < 1.0.0

File: README.md [hunk 3]
```diff
@@ -5,13 +5,12 @@
 [![npm download](https://img.shields.io/npm/dm/authmosphere.svg?style=flat-square)](https://www.npmjs.com/package/authmosphere)
 [![npm version](https://img.shields.io/npm/v/authmosphere.svg?style=flat)](https://www.npmjs.com/package/authmosphere)
 
-## Project renaming
+## Introduction
 
-The project was renamed from `lib-oauth-tooling` to `authmosphere`. In the course of this renaming versioning was restarted at `0.1.0`. Currently the change is non-breaking from a code perspective. Version `1.0.0` is soon to be released, keep track of the progess in https://github.com/zalando-incubator/lib-oauth-tooling/issues/92, this release will most likely contain breaking changes.
+Authmosphere is a JavaScript library to support some common OAuth2 use cases.
```
Comment (bzums) [inline README.md]: Maybe be more concrete than "some common"? Like:
__authmosphere__ (I would suggest to always write in lower case?) is a library to support OAuth2 workflows in JavaScript projects.

File: README.md [hunk 4]
```diff
@@ -5,13 +5,12 @@
...
 
-## Introduction
+It's implemented in TypeScript which give IDEs the ability of better autocompletion support, etc. The library itself is transpiled to JavaScript (ES6) so there is no need of a TypeScript compiler to use authmosphere in JavaScript projects.
```
Comment (bzums) [inline README.md]: Suggestion: rather describe the adavantages for the dev?

It's implemented in TypeScript which improves the development experience via implicit documentation with types and first-class IDE support. The library itself is transpiled...

Comment (Retro64) [inline README.md]: ...and advanced mock support?

Comment (bzums) [inline README.md]: Imo this part is about the advantages of TypeScript, not this lib in general.

**Comment by Retro64** [comment]
:+1:

**Comment by bzums** [comment]
👍

**Pull request merged**